task_8_9/task_8_9_5_6.py

```diff
@@ -0,0 +1,44 @@
+from utils import validate_input
+
+months = {"january": 31, "february": "28 or 29", "march": 31, "april": 30, "may": 31, "june": 30, "july": 31,
+          "august": 31, "september": 30, "october": 31, "november": 30, "december": 31}
+
+
+def input_validation_month(month):
+    while month.lower() not in months:
+        print("The entered month does not exist.")
+        month = input("Enter the month: ")
+
+    return month.lower()
+
+
+def input_validation_year(year):
+    while year < 0:
+        print('year cannot be negative')
+        year = validate_input('Enter the year')
```

Comment:
you used validate_input twice. but you could apply it only once. in this function. you don't need to use it in main